ports/raspberrypi/bindings/rp2pio/StateMachine.c

```diff
@@ -391,16 +400,18 @@ MP_DEFINE_CONST_FUN_OBJ_1(rp2pio_statemachine_stop_obj, rp2pio_statemachine_stop
 //|         :param ~circuitpython_typing.ReadableBuffer buffer: Write out the data in this buffer
 //|         :param int start: Start of the slice of ``buffer`` to write out: ``buffer[start:end]``
 //|         :param int end: End of the slice; this index is not included. Defaults to ``len(buffer)``
-//|         :param bool swap: For 2- and 4-byte elements, swap (reverse) the byte order"""
+//|         :param bool swap: For 2- and 4-byte elements, swap (reverse) the byte order
+//|         :param bool typecode: Use this typecode to set the element size. Unlike using `memoryview.cast`, this requires no allocations."""
```

Comment:
This could be explicated further. It could say that the typecode is one of B/H/L, which is used to set the element size as 1, 2, or 4 bytes. Or 1/2/4 could be passed in, since it's the length, not the actual type, that needs to be specified.

I am just thinking that why this needs to be specified requires a few sentences, and maybe better to put it here than repeat it in discord multiple times. It would be in a guide too, perhaps eventually.

"Unlike using `memoryview.cast`, this requires no allocations" -> "The size could also be set by using `memoryview.cast()` to specify the element size, but the `cast()` will cause a storage allocation."